x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.test.tsx

```diff
@@ -243,54 +207,71 @@ describe('AlertsCountPanel', () => {
             <AlertsCountPanel {...defaultProps} isExpanded={false} />
           </TestProviders>
         );
-        expect(wrapper.find('[data-test-subj="alertsCountTable"]').exists()).toEqual(false);
+        expect(wrapper.find('[data-test-subj="visualization-embeddable"]').exists()).toEqual(false);
       });
     });
   });
 });
 
-describe('when the isChartEmbeddablesEnabled experimental feature flag is enabled', () => {
+describe('renders', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     mockUseQueryToggle.mockReturnValue({ toggleStatus: true, setToggleStatus: mockSetToggle });
     mockUseIsExperimentalFeatureEnabled.mockImplementation(
       getMockUseIsExperimentalFeatureEnabled({
-        chartEmbeddablesEnabled: true,
         alertsPageChartsEnabled: false,
       })
     );
   });
 
-  it('renders LensEmbeddable', async () => {
+  it('Visualization embeddable', async () => {
```

Comment:
The verb is missing in all these `it` test cases, maybe we should change the nested "describe" to `describe('Visualization', () => ...`. And then:

- `it('should render embeddable'`
- `it('should render with the provided height'`
- `it('should render with extra actions'`
- `it('should render with extra options'`

So it will read like 
```
AlertsCountPanel
    [...]
    Query
      [...]
    toggleQuery
      [...]
    Visualization
      ✓ should render embeddable
      ✓ should render with the provided height
      ✓ should render with extra actions
      ✓ should render with extra options
```